runtime/build.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+# This script is run inside a Lambda container to compile PHP for Lambda's OS.
+# It will export a ZIP usable as a Lambda Layer.
+# See https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path
+#
+# You must specify the PHP version using the `PHP_VERSION` environment variable.
+
+set -e
+
+if [[ -z "$PHP_VERSION" ]] ; then
+    echo 'The PHP version to build must be passed in the PHP_VERSION environment variable'
+    exit 1
+fi
+
+# Disabled until we have a good bootstrap
+#cp /export/bootstrap /opt/
+cp /export/php.ini /opt/
+
+# Install dependencies
+# --releasever=2017.03: Lambda is based on 2017.03, dont' grab the latest revisions of development packages.
+# findutils: because the pecl command uses find
+# libicu and libicu-devel: because C++ & ICU are required by INTL
+yum -y --releasever=2017.03 install \
+    autoconf \
+    automake \
+    libtool \
+    bison \
+    re2c \
+    libxml2-devel \
+    openssl-devel \
+    libpng-devel \
+    libjpeg-devel \
+    curl-devel \
+    findutils \
+    php-pear \
+    libicu \
+    libicu-devel \
+    c++ \
+    gcc-c++
+
+# Compile PHP
+mkdir /tmp/bref
+cd /tmp/bref
+curl -sL https://github.com/php/php-src/archive/php-$PHP_VERSION.tar.gz | tar -zxv
+cd php-src-php-$PHP_VERSION
+./buildconf --force
+# --enable-option-checking=fatal: make sure invalid --configure-flags are fatal errors instead of just warnings
+# --enable-ftp: because ftp_ssl_connect() needs ftp to be compiled statically (see https://github.com/docker-library/php/issues/236)
+# --enable-mbstring: because otherwise there's no way to get pecl to use it properly (see https://github.com/docker-library/php/issues/195)
+# --enable-opcache-file: allows to use the `opcache.file_cache` option
+# --with-mhash: https://github.com/docker-library/php/issues/439
+./configure \
+    --enable-option-checking=fatal \
+    --enable-static=yes \
+    --enable-shared=no \
+    --with-config-file-path=/opt \
+    --disable-cgi \
+    --disable-fpm \
+    --disable-phpdbg \
+    --enable-ftp \
+    --enable-mbstring \
+    --enable-mysqlnd \
+    --enable-pcntl \
+    --enable-opcache \
+    --enable-opcache-file \
+    --enable-soap \
+    --enable-zip \
+    --with-curl \
+    --with-openssl \
+    --with-zlib \
+    --with-gd \
+    --with-pdo-mysql \
+    --with-mhash > /dev/null
+make -j 4 > /dev/null
+make install
+
+# Install extensions through pecl
+pecl install mongodb redis > /dev/null
+
+# Compile the intl extension because it cannot be installed with pecl
+cd /tmp/bref/php-src-php-$PHP_VERSION/ext/intl
+phpize > /dev/null
+./configure > /dev/null
+make > /dev/null
+make install
+
+# Copy the generated binary into /opt/bin (this directory is automatically included in the $PATH)
+# See https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path
+mkdir /opt/bin
+cp /tmp/bref/php-src-php-$PHP_VERSION/sapi/cli/php /opt/bin/
```

Comment:
And also copy php-cgi perhaps? `cp /tmp/bref/php-src-php-$PHP_VERSION/sapi/cgi/php-cgi /opt/bin/`

Reply:
Right I could if it's useful for tests right now, but at the end we want to make the layer as slim as possible (because it impacts cold starts and the limit on lambda is not that high).

Is it still useful to add it and publish a new layer or am I too late? (the benchmarks have been run it seems)